modules/op-setting-eventlistener-scc.adoc

```diff
@@ -0,0 +1,49 @@
+// This module is included in the following assemblies:
+// * secure/securing-webhooks-with-event-listeners.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='op-setting-eventlistener-scc_{context}']
+= Configuring security context for event listeners
+
+You can configure a custom security context directly in your `EventListener` definition to meet your security requirements. A custom security context can help ensure that containers run with restricted privileges and comply with {OCP} security context constraints (SCCs).
+
+.Procedure
+
+. Add a `securityContext` field under the `resources.kubernetesResource.spec.template.spec` section of your `EventListener` definition.
```

Comment:
I think this can be simplified into `Create a YAML file that defines your EventListener CR`:
. Example `EventListener` custom resource with configured security context

You can then explain what was added or modified after the example or in the annotations.

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#single-step-procedures:~:text=When%20a%20procedure%20contains%20only%20one%20step%2C%20use%20an%20unnumbered%20bullet.